lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -148,6 +150,15 @@ mkNetworkLayer j = NetworkLayer
         forM ids (getBlock j)
 
     , postTx = postMessage j
+
+    , decodeExternalTx = \payload -> do
+            case runGetOrFail getMessage (BL.fromStrict payload) of
+                Left _ ->
+                    throwE $ ErrDecodeExternalTxWrongPayload "wrongly constructed binary blob"
+                Right (_,_,msg) -> case msg of
+                    Transaction stx -> pure stx
+                    _ -> throwE $ ErrDecodeExternalTxWrongPayload "wrongly constructed binary blob"
```

Comment:
The lines are quite a bit longer than allowed by our coding standard.

Recommend the following:

```hs
    , decodeExternalTx = \payload ->
        case runGetOrFail getMessage (BL.fromStrict payload) of
            Left _ ->
                throwE $ ErrDecodeExternalTxWrongPayload
                    "wrongly constructed binary blob"
            Right (_, _, msg) ->
                case msg of
                    Transaction stx -> pure stx
                    _ ->
                        throwE $ ErrDecodeExternalTxWrongPayload
                            "wrongly constructed binary blob"
```
https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

Reply:
done